src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule_cars.java

```diff
@@ -2,8 +2,8 @@
 
 import java.net.URI;
 
-public abstract  class GoGoDuckModule_cars extends GoGoDuckModule {
-    private static final String CARS_FILENAME = "CARS2009_Australia_weekly.nc";
+public abstract class GoGoDuckModule_cars extends GoGoDuckModule {
+    private static final String CARS_FILENAME = "AODN/Australian_Government/CSIRO/Climatology/CARS/2009/eMII-product/CARS2009_Australia_weekly.nc";
```

Comment:
OK - I can see now that this works OK with the corresponding geoserver config changes - not sure how you tested it with http://data.aodn.org.au/ from the command line below (https://github.com/aodn/geoserver-build/pull/121/files#diff-49c51c5d74954153f8662c47021a5764R12) though.